pkg/config/logs/log.go

```diff
@@ -570,3 +570,21 @@ func init() {
 	seelog.RegisterCustomFormatter("ExtraTextContext", createExtraTextContext)        //nolint:errcheck
 	seelog.RegisterReceiver("syslog", &SyslogReceiver{})                              //nolint:errcheck
 }
+
+// InitConfig is used to configure logs for Datadog Agent config.
+func InitConfig(config pkgconfigmodel.Config) {
+	// If enabling log_payloads, ensure the log level is set to at least DEBUG to be able to see the logs
+	config.BindEnvAndSetDefault("log_payloads", false)
+	config.BindEnvAndSetDefault("log_file", "")
```

Comment:
A lot of these are configured in `pkg/config/system_probe.go` already

Reply:
But not all of them. My guess is we forgot to update `pkg/config/system_probe.go` when we added new config for logs. Should i remove all `log_` from `pkg/config/system_probe.go`

Reply:
My concern is that `InitSystemProbeConfig` is called in more cases than just the `init` of `pkg/config`, so maybe it would be best to do the `logs.InitConfig` in `InitSystemProbeConfig`?

Reply:
Also, the default for `log_file` is not empty string for system-probe.

Reply:
Got it. It might be better to remove `logs.InitConfig` and add missing ones in `InitSystemProbeConfig`

Reply:
> Also, the default for log_file is not empty string for system-probe. 
Am not sure if it is working as intended. We were using `Datadog` for logs , rather `SystemProbe` config.

Reply:
It gets swapped in here: https://github.com/DataDog/datadog-agent/blob/ac7747e4ecf16eea911325baff3b6b72a5f2b835/cmd/system-probe/subcommands/run/command.go#L80

Reply:
Updated the PR to add the missing config to InitSystemProbe.